sourcetools/j9constantpool/com/ibm/oti/VMCPTool/Main.java

```diff
@@ -221,85 +258,23 @@ public static void main(String[] args) throws Throwable {
 			System.exit(0);
 		}
 		
-		ObjectFactory factory = new ObjectFactory(new ConfigDirectory(configDirectory));
-		factory.initialize();
-		buildSpec = factory.getBuildSpec(buildSpecId);
-		flagDefs = factory.getFlagDefinitions();
-		
-		ConstantPool pool = parseConstantPool();
-	
+		if (cmakeCache != null){
+			if (configDirectory != null)
+				System.err.println("Ignoring option " + optionConfigDir);
```

Comment:
its preferred to use {} brackets to avoid potential problems in the future if the code is modified.